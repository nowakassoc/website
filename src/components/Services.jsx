import React from 'react';

const Services = () => (
  <section id="section-services">
    <span id="services" className="section-link" />
    <h2>Services</h2>

    <h3>Hospital, Healthcare, and Physician Practice</h3>
    <ul className="service-list">
      <li>Strategic Planning</li>
      <li>Distressed Hospital Advice and Interim Management</li>
      <li>New Business Ventures and Startup</li>
      <li>Hospital - Physician Facilitation</li>
      <li>Request for Proposal Development</li>
    </ul>

    <h3>Business Startup and Development</h3>
    <ul className="service-list">
      <li>Business Plan Development</li>
      <li>Proforma Development</li>
      <li>New business and inventor product advisory</li>
      <li>Investment Proposal Review</li>
    </ul>

    <h3>Legal</h3>
    <ul className="service-list">
      <li>Hospital, Healthcare and Physician Practice Expertise</li>
      <li>Case Review, Expert Selection, Expert Preparation</li>
      <li>Subject matter expertise and analysis of circumstances</li>
      <li>Expert witness</li>
    </ul>

    <p>References Available Upon Request</p>
  </section>
);

export default Services;
