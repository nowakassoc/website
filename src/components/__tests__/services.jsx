import React from 'react';
import { render } from 'react-testing-library';
import Services from '../Services';

it('Services renders correctly', () => {
  const tree = render(<Services />);
  expect(tree).toMatchSnapshot();
});
