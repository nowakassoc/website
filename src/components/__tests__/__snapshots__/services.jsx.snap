// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        id="section-services"
      >
        <span
          class="section-link"
          id="services"
        />
        <h2>
          Services
        </h2>
        <h3>
          Hospital, Healthcare, and Physician Practice
        </h3>
        <ul
          class="service-list"
        >
          <li>
            Strategic Planning
          </li>
          <li>
            Distressed Hospital Advice and Interim Management
          </li>
          <li>
            Board and Executive Retreat Facilitation
          </li>
          <li>
            New Business Ventures and Startup
          </li>
          <li>
            Hospital - Physician Facilitation
          </li>
          <li>
            Request for Proposal Development
          </li>
        </ul>
        <h3>
          Business Startup and Development
        </h3>
        <ul
          class="service-list"
        >
          <li>
            Business Plan Development
          </li>
          <li>
            Proforma Development
          </li>
        </ul>
        <h3>
          Legal
        </h3>
        <ul
          class="service-list"
        >
          <li>
            Hospital, Healthcare and Physician Practice Expertise
          </li>
          <li>
            Case Review, Expert Selection, Expert Preparation
          </li>
        </ul>
        <p>
          References Available Upon Request
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      id="section-services"
    >
      <span
        class="section-link"
        id="services"
      />
      <h2>
        Services
      </h2>
      <h3>
        Hospital, Healthcare, and Physician Practice
      </h3>
      <ul
        class="service-list"
      >
        <li>
          Strategic Planning
        </li>
        <li>
          Distressed Hospital Advice and Interim Management
        </li>
        <li>
          Board and Executive Retreat Facilitation
        </li>
        <li>
          New Business Ventures and Startup
        </li>
        <li>
          Hospital - Physician Facilitation
        </li>
        <li>
          Request for Proposal Development
        </li>
      </ul>
      <h3>
        Business Startup and Development
      </h3>
      <ul
        class="service-list"
      >
        <li>
          Business Plan Development
        </li>
        <li>
          Proforma Development
        </li>
      </ul>
      <h3>
        Legal
      </h3>
      <ul
        class="service-list"
      >
        <li>
          Hospital, Healthcare and Physician Practice Expertise
        </li>
        <li>
          Case Review, Expert Selection, Expert Preparation
        </li>
      </ul>
      <p>
        References Available Upon Request
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
